test/e2e/_allocation.js

```diff
@@ -0,0 +1,28 @@
+import { t } from 'testcafe'
+
+import { allocationDetailsPage, allocationCriteriaPage, page } from './pages/'
+
+async function createAllocation() {
```

Comment:
We actually wanted to move away from this pattern of `_allocation`/`_move` file. The move example is a bit of a cross between helpers and a page object right now and I think we need to split it up a little moving forward. 

It would be good not to use this pattern for allocations as it will just introduce more tech debt. These methods look like they would fit into the page object model so if we need them to live somewhere, they should live within the page objects.